tests/cases/text/control_keys_input.slint

```diff
@@ -0,0 +1,39 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 100phx;
+    height: 100phx;
+    ti := TextInput { 
+    }
+
+    property <bool> input_focused: ti.has_focus;
+    property <string> text <=> ti.text;
+}
+
+/*
+```rust
+
+const SHIFT_CODE: char = '\u{0010}';
+const SHIFTR_CODE: char = '\u{0015}';
+const ALT_CODE: char = '\u{0012}';
+const ALTGR_CODE: char = '\u{0013}';
+const CONTROL_CODE: char = '\u{0011}';
+const CONTROLR_CODE: char = '\u{0016}';
+const META_CODE: char = '\u{00E0}';
+const METAR_CODE: char = '\u{00E1}';
+
+let instance = TestCase::new();
+slint_testing::send_mouse_click(&instance, 5., 5.);
+assert!(instance.get_input_focused());
+assert!(instance.get_text().is_empty());
+
+for code in [SHIFT_CODE, SHIFTR_CODE, ALT_CODE, ALTGR_CODE, CONTROL_CODE, CONTROLR_CODE, META_CODE, METAR_CODE] {
+    slint_testing::send_keyboard_char(&instance, code, true);
+    assert!(instance.get_text().is_empty());
+    slint_testing::send_keyboard_char(&instance, code, false);
+    assert!(instance.get_text().is_empty());
+}
```

Comment:
maybe also send a valid character at the end to make sure the test work?

Reply:
Sure. We do that in another test (keyboard_modifiers.slint), but I can also add it here, too.

Reply:
right, this is not strictly required, but it makes the test more robust.

Imagine that we do an intentional change, possibly breaking that cause the input to not be processed. (example, we decide that elements should now be centered, or we add a default padding to the window, or whatnot) then the test in keyboard_modifiers.slint will fail, and we will fix that test, but this test will nt fail, so it will not be fixed, and when we break the control code again, the test will not fail.  
This is a bit far fetched and I don't think this scenario will happen. But I just think it's good practice to have robust tests.

The test could also easily test other keys like F1, PrintSceern, ScrollLock, Enter, Play, PageDown, and such.